### ISSUE TITLE
Replace mo:wikipedia

### ISSUE THREAD

**Issue by acka47** [hbz/lobid-organisations#336]
Taking a look at the rdfs:domain of mo:wikipedia we should use another property in the context of lobid-organisations. From the Music Ontology:

```
mo:wikipedia
    mo:level "1" ;
    a owl:ObjectProperty ;
    rdfs:comment """
        Used to link an work, an expression of a work, a manifestation of a work, 
        a person, an instrument or a musical genre to its corresponding WikiPedia page. 
        The full URL should be used, not just the WikiName.
    """ ;
    rdfs:domain [
        a owl:Class ;
        owl:unionOf (frbr:Work
            frbr:Expression
            frbr:Manifestation
            frbr:Item
            foaf:Agent
            mo:Instrument
            mo:Genre
        )
    ] ;
    rdfs:isDefinedBy <http://purl.org/ontology/mo/> ;
    rdfs:label "wikipedia" ;
    rdfs:range foaf:Document ;
    rdfs:subPropertyOf foaf:isPrimaryTopicOf ;
    vs:term_status "stable" .
```

**Comment by acka47** [comment]
As wa want to add wikidata links (see #320), we should also think about removing the wikpedia links completely.

**Comment by ChristophEwertowski** [comment]
Possible properties: http://purl.org/dc/terms/description or https://schema.org/description.

**Comment by acka47** [comment]
This will be solved in #320 with the removal of wikipedia links.
